### DISCUSSION TITLE
Reachability system gestures conflicts

### DISCUSSION THREAD

**Discussion by Alisher9** [LeoNatan/LNPopupController#507] [General] [answered]
In my app I have swipeDownGesture for popupItem above tabbarController and it conflicts with system gestures of iPhone(Reachability). Is it possible to disable these system gestures in Xcode?

**Comment by Alisher9** [comment]
Also, if I change the swipe direction to left/right, it doesn't dismiss popupBar. Don't you know why?

**Comment by LeoNatan** [comment, marked as answer]
You cannot disable system gesture recognizers. You should not fight the system. Swiping up/down in the bottom part of the screen is probably not a good idea. Not sure why left/right doesn't work, but I wouldn't recommend it either, as those might get caught in the app switcher gestures.
